tests/Jellyfin.Server.Implementations.Tests/Library/IgnorePatternsTests.cs

```diff
@@ -0,0 +1,19 @@
+using Emby.Server.Implementations.Library;
+using Xunit;
+
+namespace Jellyfin.Server.Implementations.Tests.Library
+{
+    public class IgnorePatternsTests
+    {
+        [Theory]
+        [InlineData("/media/small.jpg", true)]
+        [InlineData("/media/movies/#Recycle/test.txt", true)]
+        [InlineData("/media/movies/#recycle/", true)]
+        [InlineData("thumbs.db", true)]
+        [InlineData(@"C:\media\movies\movie.avi", false)]
```

Comment:
pls add tests on Unix hidden files, like
```suggestion
        [InlineData(@"C:\media\movies\movie.avi", false)]
        [InlineData("/media/.hiddendir/file.mp4", false)]
        [InlineData("/media/dir/.hiddenfile.mp4", true)]
```